man/jail.conf.5

```diff
@@ -146,6 +146,12 @@ Ensure syslog or the program that generates the log file isn't configured to com
 .B logencoding
 encoding of log files used for decoding. Default value of "auto" uses current system locale.
 .TP
+.B banaction
+default banning action (iptables-multiport) for all jails specified in the \fI[DEFAULT]\fR section.
```

Comment:
Yes, but I meant: the banning action (for all jails) specified in the `[DEFAULT]` section.
Not the jails in default section. My bad english :(